assets/js/index.js

```diff
@@ -0,0 +1,901 @@
+
+	// Check for toggle/untoggle on screen resize
+	resizeCheck: function () {
+		if (document.querySelectorAll('*[data-untoggle-above], *[data-untoggle-below], *[data-toggle-above], *[data-toggle-below]').length) {
+			window.addEventListener('resize', function () {
+				var winWidth = window.innerWidth,
+					toggles = document.querySelectorAll('.toggle');
+
+				toggles.forEach(function (toggle) {
+					var unToggleAbove = toggle.dataset.untoggleAbove,
+						unToggleBelow = toggle.dataset.untoggleBelow,
+						toggleAbove = toggle.dataset.toggleAbove,
+						toggleBelow = toggle.dataset.toggleBelow;
+
+					// If no width comparison is set, continue
+					if (!unToggleAbove && !unToggleBelow && !toggleAbove && !toggleBelow) {
+						return;
+					}
+
+					// If the toggle width comparison is true, toggle the toggle
+					if (
+						(((unToggleAbove && winWidth > unToggleAbove) ||
+							(unToggleBelow && winWidth < unToggleBelow)) &&
+							toggle.classList.contains('active')) ||
+						(((toggleAbove && winWidth > toggleAbove) ||
+							(toggleBelow && winWidth < toggleBelow)) &&
+							!toggle.classList.contains('active'))
+					) {
+						toggle.click();
+					}
+				});
+			});
+		}
+	},
+
+	// Close toggle on escape key press
+	untoggleOnEscapeKeyPress: function () {
+		document.addEventListener('keyup', function (event) {
+			if (event.key === 'Escape') {
+				document.querySelectorAll('*[data-untoggle-on-escape].active').forEach(function (element) {
+					if (element.classList.contains('active')) {
+						element.click();
+					}
+				});
+			}
+		});
+	},
+
+}; // twentytwenty.toggles
+
+window.wp.domReady(function () {
+	twentytwenty.intervalScroll.init();			// Check for scroll on an interval
+	twentytwenty.resizeEnd.init();				// Trigger event at end of resize
+	twentytwenty.toggles.init();				// Handle toggles
+	twentytwenty.coverModals.init();			// Handle cover modals
+	twentytwenty.intrinsicRatioVideos.init();	// Retain aspect ratio of videos on window resize
+	twentytwenty.smoothScroll.init();			// Smooth scroll to anchor link or a specific element
+	twentytwenty.scrollLock.init();				// Scroll Lock
+	twentytwenty.mainMenu.init();				// Main Menu
+	twentytwenty.focusManagement.init();		// Focus Management
+	twentytwenty.dynamicScreenHeight.init();	// Dynamic Screen Height
+});
+
+
+
+/*	-----------------------------------------------------------------------------------------------
+	Helper functions
+--------------------------------------------------------------------------------------------------- */
+
+/* Toggle an attribute ----------------------- */
+
+function twentytwentyToggleAttribute(element, attribute, trueVal, falseVal) {
+	if (trueVal === undefined) {
+		trueVal = true;
+	}
+	if (falseVal === undefined) {
+		falseVal = false;
+	}
+	if (element[attribute] !== trueVal) {
+		element.setAttribute(attribute, trueVal);
+	} else {
+		element.setAttribute(attribute, falseVal);
+	}
+}
+
+function getQueryStringValue() {
```

Comment:
Most of these functions should be prefixed (from bellow), they are in the the global scope